frontend_lib/src/component/CardPopup/CardPopup.styl

```diff
@@ -23,7 +23,6 @@
     display flex
     justify-content flex-end
     padding 5px 15px 5px 5px
-    cursor pointer
```

Comment:
Do the close button still have a pointer cursor?

Reply:
Yes